processor/workers.go

```diff
@@ -693,6 +693,13 @@ func fileProcessorWorker(input chan *FileJob, output chan *FileJob) {
 					continue
 				}
 
+				if NoLarge && res.Lines >= LargeLineCount {
+					if Verbose {
+						printWarn(fmt.Sprintf("skipping large file due to line length: %s", res.Location))
+					}
+					continue
+				}
+
```

Comment:
Maybe it's worth moving this into `CountStats` and `return` as soon as the line limit is reached? It would save a least some processing rather than waiting until after the file is fully processed.

Reply:
I did consider that... I would hope usually the byte limit kicks in first. The outside part is still needed, just it exits earlier. So you get something like

Line 485 in workers.go
```
                // This means the end of processing the line so calculate the stats according to what state
		// we are currently in
		if fileJob.Content[index] == '\n' || index >= endPoint {
			fileJob.Lines++

                        if NoLarge && fileJob.Lines >= LargeLineCount {
                            fileJob.Content = nil
                            return
                        }
 
			switch currentState {
```

Worth it? Not sure. I don't know what amount of files that pass the byte limit that would hit this one to make it worth it. Trivial to implement though.

Reply:
Added anyway.